post/release-0-2.qmd

```diff
@@ -14,39 +21,123 @@ since the last specification release.
+    "crs_type": "wkt2:2019",
+  }
+  ```
+
+- Authority code example:
+
+  ```json
+  {
+      "crs": "EPSG:32620",
+      "crs_type": "authority_code",
+  }
+  ```
+
+  Note that this should only be used as a last resort. Producers should prefer writing a complete description of the CRS.
+
+- SRID example:
+
+  ```json
+  {
+      "crs": "32620",
+      "crs_type": "srid",
+  }
+  ```
+
+  Note that this should only be used as a last resort for database drivers or readers that have no other option.
+
+- Opaque CRS example:
+
+  ```json
+  {
+      "crs": 'PROJCS["WGS 84 / UTM zone 20N",GEOGCS["WGS 84",DATUM["WGS_1984",SPHEROID["WGS 84",6378137,298.257223563,AUTHORITY["EPSG","7030"]],AUTHORITY["EPSG","6326"]],PRIMEM["Greenwich",0,AUTHORITY["EPSG","8901"]],UNIT["degree",0.0174532925199433,AUTHORITY["EPSG","9122"]],AUTHORITY["EPSG","4326"]],PROJECTION["Transverse_Mercator"],PARAMETER["latitude_of_origin",0],PARAMETER["central_meridian",-63],PARAMETER["scale_factor",0.9996],PARAMETER["false_easting",500000],PARAMETER["false_northing",0],UNIT["metre",1,AUTHORITY["EPSG","9001"]],AXIS["Easting",EAST],AXIS["Northing",NORTH],AUTHORITY["EPSG","32620"]]',
```

Comment:
Using `'` quotes means that the internal `"` don't need to be escaped `\"`. I think it's fine for this code section to be "json like" instead of literally json text

Reply:
I didn't want to use `Auth:code` because in that case they should be applying the relevant `crs_type` (except in the case when they don't _know_ it's an auth:code, but I didn't want to get into all those details right here)